### DISCUSSION TITLE
getShopReceipts: shipping_method

### DISCUSSION THREAD

**Discussion by KartikLan** [etsy/open-api#956] [Q&A]
shipping_method is removed from receipt object and added to transaction object. But API v3 is returning null shipping method for all transactions.

**Comment by Gareth064** [comment]
Hey @KartikLan  👋

When ShippingMethod is null, it means "Standard Shipping" has been chosen by the customer or another way to look at it is, It will contain data when a shipping upgrade is chosen.

**Comment by fannigurt** [comment]
I see in my receipts:
in a transaction in "transactions" such fields:
```
"shipping_method": null,
"shipping_upgrade": "UPS 3 Day Select"
```
for upgrade, and
```
"shipping_method": null,
"shipping_upgrade": null,
```
for the "standard" option.

Hope this will help you.
Also it could be that this are specific settings of my merchant, so please double check with your different receipts and transactions.
